protocol/requests/edit_message.go

```diff
@@ -12,11 +12,12 @@ var ErrEditMessageInvalidID = errors.New("edit-message: invalid id")
 var ErrEditMessageInvalidText = errors.New("edit-message: invalid text")
 
 type EditMessage struct {
-	ID                 types.HexBytes                   `json:"id"`
-	Text               string                           `json:"text"`
-	ContentType        protobuf.ChatMessage_ContentType `json:"content-type"`
-	LinkPreviews       []common.LinkPreview             `json:"linkPreviews"`
-	StatusLinkPreviews []common.StatusLinkPreview       `json:"statusLinkPreviews"`
+	ID                 types.HexBytes             `json:"id"`
+	Text               string                     `json:"text"`
+	LinkPreviews       []common.LinkPreview       `json:"linkPreviews"`
+	StatusLinkPreviews []common.StatusLinkPreview `json:"statusLinkPreviews"`
+	// Deprecated: The ContentType cannot be edited
+	ContentType protobuf.ChatMessage_ContentType `json:"content-type"`
```

Comment:
This field is only used now in a test. Wouldn't it be safe to remove the field from the struct, or would that cause the endpoint using the request struct to blow up?

Reply:
I assumed that if clients still use it it would cause the request to fail, but I haven't tested. I'll give it a go and report.

Reply:
I just tested and the request accepts "unknown" params. So even if mobile still send ContentType, it would still work. I can remove it completely then

Reply:
After @churik confirms the bug is gone and https://github.com/status-im/status-mobile/issues/21540 is closed we will have an issue to stop sending the parameter as well @jrainville.